private-templates-service/client/src/store/page/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface PageState {
+  currentPage: string;
+}
+
+// Action Types
+export const NAVIGATION = "NAVIGATION";
+
+// Actions 
+export interface PageAction{
+  type: typeof NAVIGATION;
+  text: string;
+  currentPage: string;
```

Comment:
currentPageTitle

Reply:
I changed all instances of currentPage -> currentPageTitle to make it more clear that the title page is rendering